sourmash/sbt.py

```diff
@@ -312,6 +312,13 @@ def gather(self, query, *args, **kwargs):
         # use a tree search function that keeps track of its best match.
         search_fn = GatherMinHashes().search
 
+        leaf = next(iter(self.leaves()))
+        tree_mh = leaf.data.minhash
+        scaled = tree_mh.scaled
```

Comment:
Oh, the classic "what queries does this tree support" trick =P

Reply:
<sigh>